.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,68 @@
+name: Brazillian Orders CI/CD Pipeline to EC2 
+
+on:
+  push:
+    branches: [ "orders_api_deployment" ]
```

Comment:
Nice, a better practice is to make this main branch but this works too. So you main branch is protected from regular pushes

Reply:
okay.. noted

Reply:
Sorry, Does it mean it being protected from regular pushes made by me or from pull requests made by collaborators ?

Reply:
Alright